test/system/autocomplete_component_test.rb

```diff
@@ -67,22 +83,26 @@ def test_remote_autocomplete
   def test_empty_state
     with_preview("forms/autocomplete_component/empty_state")
 
-    find(".Polaris-TextField__Input").click
-    find(".Polaris-TextField__Input").set "Unknown"
+    open_autocomplete
+    type_text "Unknown"
     assert_selector ".Polaris-Autocomplete__EmptyState", text: "Could not find any results"
   end
 
   def test_event_handler
     with_preview("forms/autocomplete_component/event_handler")
 
-    # Open autocomplete
-    find(".Polaris-TextField__Input").click
+    open_autocomplete
     assert_selector ".Polaris-OptionList-Option", text: "Rustic"
 
     # Select Vinyl
     accept_alert "Selected vinyl" do
       find(".Polaris-OptionList-Option", text: "Vinyl").click
     end
-    assert_selector ".Polaris-OptionList-Option--select", text: "Vinyl"
   end
+
+  private
+
+  def autocomplete_input_field = find(".Polaris-TextField__Input")
+  def open_autocomplete = autocomplete_input_field.click
+  def type_text(text) = autocomplete_input_field.set(text)
```

Comment:
Let me know if these methods make sense and if you like this endless syntax.

Reply:
This won't pass our code style guide ([Standard](https://github.com/testdouble/standard)). Let's stick with the default syntax here for consistency.